### PR TITLE
Total backed debt consistency

### PR DESCRIPTION
**Pull request by kmbarry1**
Add check that fundamental equation of dai is not violated.

### CODE REVIEW

**Review by iamchrissmith — APPROVED** [review]

File: kmcd-props.md [hunk 1]
```diff
@@ -21,12 +21,15 @@ Measurables
 ### Measure Event
 
 ```k
-    syntax Event ::= Measure ( debt: Rat , controlDai: Map , potChi: Rat , potPie: Rat )
+    syntax Event ::= Measure ( debt: Rat , controlDai: Map , potChi: Rat , potPie: Rat , sumOfScaledArts: Rat, vice: Rat )
  // ------------------------------------------------------------------------------------
```
Comment (ehildenb) [inline kmcd-props.md]: formatting (extend the `---`)

File: kmcd-props.md [hunk 2]
```diff
@@ -132,6 +135,18 @@ Total dai of all users = CDP debt for all users and gem + system debt (vice)
     rule sumOfAllUserDebt(.Map, _, SUM) => SUM
 ```
 
+Total backed debt (sum over each CDP's art times corresponding ilk's rate)
+
+```k
+    syntax Rat ::= calcSumOfScaledArts(Map, Map) [function]
+                   | calcSumOfScaledArtsAux(List, Map, Map, Rat) [function]
+ // ------------------------------------------------------------------------------------
```
Comment (ehildenb) [inline kmcd-props.md]: formatting (line up the `|` with the `=` on previous line, shorten the `---`)

File: kmcd-props.md [hunk 3]
```diff
@@ -207,6 +223,16 @@ A default `owise` rule is added which leaves the FSM state unchanged.
          <properties> ... VFSMID |-> (VFSM => derive(VFSM, E)) ... </properties>
 ```
 
+### Total Backed Debt Consistency
+
+Vat.debt minus Vat.vice should equal the sum over all ilks and CDP accounts of the CDP's art times the ilk's rate.
+
+```k
+    syntax ViolationFSM ::= "totalBackedDebtConsistency"
+ // --------------------------------------------------------------------
```
Comment (ehildenb) [inline kmcd-props.md]: formatting

File: kmcd-props.md (hunk 1 above)
Comment (kmbarry1) [reply kmcd-props.md]: Done.

File: kmcd-props.md (hunk 2 above)
Comment (kmbarry1) [reply kmcd-props.md]: Done.

File: kmcd-props.md (hunk 3 above)
Comment (kmbarry1) [reply kmcd-props.md]: Done.

**Pull request merged**